devices/fields/spi/spi_v3.yaml

```
@@ -0,0 +1,21 @@
+_include:
+  - spi_v3_base.yaml
+  - spi_v3_i2s.yaml
+
+CFG1:
+  UDRDET:
+    StartOfFrame: [0, Underrun is detected at begin of data frame]
+    EndOfFrame: [1, Underrun is detected at end of last data frame]
+    StartOfSlaveSelect: [2, Underrun is detected at begin of active SS signal]
+  UDRCFG:
+    RepeatTransmitted: [2, Slave repeats last transmitted data frame]
+CR2:
+  TSER: [0, 65535]
+IER:
+  TSERFIE:
+    Masked: [0, TSER loaded interrupt masked]
+    NotMasked: [1, TSER loaded interrupt not masked]
```

What about enabled/disabled instead of masked/notmasked?
```suggestion
    Disabled: [0, TSERF interrupt disabled]
    Enabled: [1, TSERF interrupt enabled]
```
From [RM0433](https://www.st.com/resource/en/reference_manual/rm0433-stm32h742-stm32h743753-and-stm32h750-value-line-advanced-armbased-32bit-mcus-stmicroelectronics.pdf) page 2224:

> Bit 10 TSERFIE: additional number of transactions reload interrupt enable
>   0: TSERF interrupt disabled
>   1: TSERF interrupt enabled

---

Different enums for each field are not needed here. Replaced with:
```yaml
  "*IE":
    Disabled: [0, Interrupt disabled]
    Enabled: [1, Interrupt enabled]
```

---

nice